test/browser/browser.sh

```diff
@@ -62,6 +62,12 @@ systemctl start firewalld
 firewall-cmd --add-service=cockpit --permanent
 firewall-cmd --add-service=cockpit
 
+if grep -Eq 'PLATFORM_ID==.*(f35)' /etc/os-release; then
```

Comment:
This [did not work](http://artifacts.dev.testing-farm.io/1554aa3c-c62e-41b1-89b3-940d9a2cf4c3/). os-release has no double `==`, but it should work with a single one.

Reply:
Fixe (FPed your branch)